spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -4072,6 +4072,14 @@ def _handle_completions(self):
                 self._last_pressed_key = None
                 return
 
+        if key == Qt.Key_Backspace:

```

Comment:
@steff456, please add a comment above this line to explain why this `if` is necessary here.